src/test/kotlin/com/hibob/academy/dao/OwnerDaoTest.kt

```diff
@@ -26,53 +26,36 @@ class OwnerDaoTest @Autowired constructor(private val sql: DSLContext){
 
     @Test
     fun `create a new owner that doesn't exist in the database`() {
-        val ownerName = "Noy"
-        val ownerCompanyId = companyId
-        val ownerEmployeeId = "123"
-        val ownerId = ownerDao.createOwner(ownerName, ownerCompanyId, ownerEmployeeId)
-        val expected = listOf(OwnerData(ownerId, ownerName, ownerCompanyId, ownerEmployeeId))
-        assertEquals(expected, ownerDao.getOwner(ownerCompanyId))
+        val owner = OwnerCreationRequest("Noy", companyId, "123")
+        val ownerId = ownerDao.createOwner(owner)
+        val expected = listOf(OwnerData(ownerId, owner.name, owner.companyId, owner.employeeId))
+        assertEquals(expected, ownerDao.getAllOwners(companyId))
     }
 
     @Test
     fun `create a new owner that exists in the database`() {
```

Comment:
reate a new owner that exists in the database - they are not identical - the employee id is